CircuitPy/code.py

```diff
@@ -28,5 +42,18 @@
 
 # Main loop will read the range and print it every second.
 while True:
-    print("{0}, {0}".format(vl53_1.range, vl53_2.range))
-    time.sleep(1.0)
\ No newline at end of file
+    try:
+        sensor1Range = vl53_1.range
+    except:
+        sensor1Range = -1
+        
+        
+    try:
+        sensor2Range = vl53_2.range
+        
+    except:
+        sensor2Range = -1
+    print("{0}, {1},".format(sensor1Range, sensor2Range))
+    uart.write(bytearray("{0}, {1},\r\n".format(sensor1Range, sensor2Range)))
```

Comment:
Why include the carriage return (\r)? Doesn't seem like the roboRIO code expects it